src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,32 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
     public User register(User user) {
-        return null;
+        if (user.getAge() < MIN_AGE) {
+            throw new RuntimeException("The user must be older than " + MIN_AGE);
+        }
+        if (user.getPassword().length() < MIN_PASSWORD_LENGTH) {
+            throw new RuntimeException("The password length must be longer than "
+                    + MIN_PASSWORD_LENGTH);
+        }
+
+        if (user.getLogin() == null) {
+            throw new RuntimeException("The login can not be null!");
+        }
+
+        if (user.getPassword() == null) {
+            throw new RuntimeException("The password can not be null!");
+
+        }
```

Comment:
Is it possible to check `user.getPassword().length() < MIN_PASSWORD_LENGTH` before `user.getPassword() == null` ??